app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
I don't know of anyone asking for it.  If we are not allowing delete based on assignment, it might be nice to show that column in the list and summary views of the rates though.  Could even disable the `Remove` button on the summary screen.